stdlib/src/blackbox_fallbacks/blake2s.rs

```diff
@@ -197,18 +196,22 @@ fn blake2s_compress(
     let mut m = Vec::new();
     let mut v = Vec::new();
 
-    for i in 0..16 {
-        let mut mi_bytes = input.get(i * 4..i * 4 + 4).unwrap().to_vec();
-        mi_bytes.reverse();
-        let (mi, extra_gates, updated_witness_counter) =
-            UInt32::from_witnesses(&mi_bytes, num_witness);
-        new_gates.extend(extra_gates);
-        m.push(mi[0]);
-        num_witness = updated_witness_counter
+    unroll! {
+        for i in 0..16 {
+            let mut mi_bytes = input.get(i * 4..i * 4 + 4).unwrap().to_vec();
+            mi_bytes.reverse();
+            let (mi, extra_gates, updated_witness_counter) =
+                UInt32::from_witnesses(&mi_bytes, num_witness);
+            new_gates.extend(extra_gates);
+            m.push(mi[0]);
+            num_witness = updated_witness_counter
+        }
     }
 
-    for i in 0..8 {
-        v.push(state.h[i]);
+    unroll! {
```

Comment:
This is not needed imo, we don't really care about performance of these fallback functions right now